syz-manager/cover.go

```diff
@@ -112,7 +112,49 @@ func parseFile(fn string) ([][]byte, error) {
 	return lines, nil
 }
 
+func getVmOffset(vmlinux string)(uint32, error) {
+	cmd := exec.Command("readelf", "-SW", vmlinux)
```

Comment:
If a process terminates, output is bounded and you just need the output, then a simpler alternative is:
```
  out, err := exec.Command("readelf", "-SW", vmlinux).CombinedOutput()
  if err != nil {
    return 0, fmt.Errorf("readelf failed: %v\n%s", err, out)
  }
  s := bufio.NewScanner(bytes.NewReader(out))
  ...
```

CombinedOutput starts the process, waits for completion and collects output.